src/Umbraco.Web.Common/Formatters/IgnoreRequiredAttributesResolver.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Reflection;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
+
+namespace Umbraco.Web.Common.Formatters
+{
+    public class IgnoreRequiredAttributesResolver : DefaultContractResolver
```

Comment:
Can we create tests for this new resolver?

Reply:
I added one :)